WebApp/autoreduce_webapp/selenium_tests/tests/test_accessibility_statement_page.py

```diff
@@ -0,0 +1,28 @@
+from selenium_tests.pages.accessibility_statement_page import AccessibilityStatementPage
+from selenium_tests.tests.base_tests import NavbarTestMixin, BaseTestCase, \
+    FooterTestMixin
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+class TestAccessibilityStatementPage(NavbarTestMixin, BaseTestCase, FooterTestMixin):
+    """
+    Test cases for the accessibility statement page
+    """
+    def setUp(self) -> None:
+        """
+        Sets up the AccessibilityStatementPage object
+        """
+        super().setUp()
+        self.page = AccessibilityStatementPage(self.driver)
+        self.page.launch()
+
+    def test_accessibility_statement_contents_appears(self):
+        """
+        Test that the <div> #accessibility-statement-contents is visible and contains text
+        """
+        WebDriverWait(self.driver,
```

Comment:
Are these waits necessary since the statement is loaded by default and not by JavaScript or some other action and the driver is setup with an implicit wait to begin with. 
i.e.
```python
self.assertTrue(self.page.get_accessibility_statement_contents_element().is_displayed())
```

Or move the check to the page and more succinctly:

```python
self.assertTrue(self.page.is_accessibility_statement_visible())
```

Reply:
Not sure why I used `WebDriverWait`. I'll add those suggestions. Thanks